common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -315,7 +316,22 @@ main() {
             ;;
         "start_wait")
             start_function
-            wait -n
+
+            # Store PIDs of the Controller and Agents
+            PREV_PIDS=$(pgrep "beerocks_(agent|controller)")
+            CURR_PIDS=$PREV_PIDS
```

Comment:
For consistency with below, I'd write

```
CURR_PIDS="$(pgrep ...)"
PREV_PIDS="$CURR_PIDS"
```

Also the right-hand side really should be quoted, like I did above.

Reply:
What are the benefits of quoting in this case?

Reply:
Not quoting the right hand side of an assignment is also a bashism. With some shells, you'll get CURR_PIDS being set to the first pid, then the shell will probably try to execute the next one as if it was a command.

Reply:
Fixed